### PR TITLE
Update 01_network-nervous-system-nns.card.md

### PR DESCRIPTION
**Pull request by bjoernek**
Update of the "abstract card" for the NNS.

### CODE REVIEW

File: how-it-works/5_governance/01_network-nervous-system-nns.card.md [hunk 1]
```diff
@@ -6,6 +6,6 @@ title: Network Nervous System
...
+The Internet Computer is a *decentralized* system run by many independent machines. These machines have to agree on certain facts, for example on what machines are part of the system, what program they should run, and when this program should be upgraded. On the Internet Computer these decisions are made by an open, tokenized governance system that is called the *Network Nervous System (NNS)*. The NNS is one of the largest decentralized autonomous organizations (DAOs). Anyone can become a participant of the NNS and contribute to decisions by staking ICP tokens.
 
-[Go deeper](/how-it-works/network-nervous-system-nns/)
+[Go deeper](https://internetcomputer.org/nns/)
```
Comment (sesi200) [inline how-it-works/5_governance/01_network-nervous-system-nns.card.md:11]:
```suggestion
[Go deeper](/nns/)
```
bitte nicht volle URLs, sonst merkt CI nicht wenn der link nicht mehr funktioniert

**Review by sesi200 — CHANGES_REQUESTED** [review]

**Review by Dfinity-Bjoern — COMMENTED** [review]
I think the text is a bit misleading right now – in particular the part on "those nodes have to agree on facts". To me, it seems a reader would then expect to learn about consensus, so it sets the wrong expectation.

Here are my suggestions:
- remove the part on "agree", maybe rather say that all nodes must "know" the topology (which nodes contribute to which subnets etc.) and the protocol versions
- probably it's better to talk about the version of the protocol (not program) that the nodes use

File: how-it-works/5_governance/01_network-nervous-system-nns.card.md [hunk 2]
```diff
@@ -6,6 +6,6 @@ title: Network Nervous System
...
 
-The Network Nervous System (NNS) is the open algorithmic system governing the Internet Computer blockchain. Its most notable innovations include its ability to upgrade the Internet Computer protocol and software running on node machines, onboard new node providers, add node machines into the blockchain network, and create new subnet blockchains to increase capacity. Anyone can participate in the NNS by staking ICP tokens into one or more neurons. Neurons can submit proposals and decide to adopt or reject them. The NNS implements liquid democracy: neurons can follow other neurons a delegate voting power.
+The Internet Computer is a *decentralized* system run by many independent machines. These machines have to agree on certain facts, for example on what machines are part of the system, what program they should run, and when this program should be upgraded. On the Internet Computer these decisions are made by an open, tokenized governance system that is called the *Network Nervous System (NNS)*. The NNS is one of the largest decentralized autonomous organizations (DAOs). Anyone can become a participant of the NNS and contribute to decisions by staking ICP tokens.
```
Comment (LaraAS) [inline how-it-works/5_governance/01_network-nervous-system-nns.card.md:9]:
```suggestion
The Internet Computer is a *decentralized* system run by many independent machines. These machines all have to know certain facts, for example to which subnet they contribute (the topology), which protocol version they should run, and when they should upgrade to a new protocol version. On the Internet Computer these decisions are made by an open, tokenized governance system that is called the *Network Nervous System (NNS)*. The NNS is one of the largest decentralized autonomous organizations (DAOs). Anyone can become a participant of the NNS and contribute to decisions by staking ICP tokens.
```

**Review by LaraAS — COMMENTED** [review]
@Dfinity-Bjoern thanks for the comments! 
how about the suggested change?
I wanted to avoid using too many technicals terms such as "nodes" etc, but still address your feedback.

**Review by Dfinity-Bjoern — APPROVED** [review]
That's a good start now, thanks @bjoernek and @LaraAS

**Review by sesi200 — APPROVED** [review]

**Pull request merged**